src/commands/reputation/giveReputation.test.ts

```diff
@@ -141,6 +116,39 @@ describe('giveRep', () => {
 
     expect(mockCreateUpdateUser).toHaveBeenCalledTimes(1);
     expect(mockUpdateRep).toHaveBeenCalledTimes(1);
-    expect(replyMock).toHaveBeenCalledTimes(1);
+    expect(replyMock).toHaveBeenCalledTimes(0);
+    expect(sendMock).toHaveBeenCalledTimes(1);
+  });
+
+  it('should give rep multiple times if mentions more than one user', async () => {
+    const mockUsers = new Collection<string, User>();
+    mockUsers.set('0', { id: '0', bot: false } as User);
+    mockUsers.set('1', { id: '1', bot: true } as User);
+    mockUsers.set('2', { id: '2', bot: false } as User);
+    mockUsers.set('3', { id: '3', bot: false } as User);
```

Comment:
Should we test the case of repeated mentions? E.g. `thanks @sam, @sam, @sam`?

Reply:
This does not need to be the case, since Discord's mentioned user are all unique and won't account for duplicate mentions.
![Screen Shot 2022-03-14 at 1 53 32 am](https://user-images.githubusercontent.com/4188758/158065440-f3346cc7-250b-4459-8f9d-a794a8b5e58f.png)
![Screen Shot 2022-03-14 at 1 53 45 am](https://user-images.githubusercontent.com/4188758/158065452-ca35b53c-ceaf-450b-a51c-5ff72be13a59.png)